src/support_sphere_py/src/support_sphere/models/enums/operational_status.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class OperationalStatus(Enum):
+    EMERGENCY = ("emergency", "Total Emergency Mode")
+    TEST = ("test", "Test Emergency Mode")
+    NORMAL = ("normal", "We're all good capn'")
```

Comment:
```suggestion
    NORMAL = ("normal", "Normal Operation Mode")
```

Haha that description was suppose to be a joke 😄